app/views/users/index.html.erb

```diff
@@ -1,3 +1,16 @@
-<div>
-     <h1> Here is a list of all users <h1>
-<div />
\ No newline at end of file
+<section>
+  <ul class='user-container'>
+    <% @users.each do |user| %>
+      <li class='user'>
+        <div class='user-photo'>
+          <img src='<%= user.photo %>' alt='<%= user.name %> thumbnail' />
+        </div>
+        <div class='user-name'>
+          <h2><%= link_to user.name, user_path(user) %></h2>
+          <p>Number of Posts: <%= user.posts_counter %></p>
+        </div>
+      </li>
+    <% end %>
+  </ul>
```

Comment:
[OPTIONAL ]
- A better approach of rendering `users` to the corresponding views page rather than using `@users.each`, is to make use of `partials`. That can achieved using the code below:
```
# app/views/users/_user.html.erb
     HTML Code Here displaying
     user.photo, user.name and user.posts_counter
```
```
    # app/views/users/index.html.erb
    <%= render @users %>
```